iam_openapi_spec.yml

```diff
@@ -994,6 +999,22 @@ paths:
           $ref: '#/components/responses/ErrorResponse'
       security: [ ]
 
+  /verifyEmail:
+    post:
+      tags:
+        - Passcode Management
+      summary: Verify email
+      description: Verify email during account opening
+      operationId: verifyEmail
+      requestBody:
+        $ref: '#/components/requestBodies/VerifyEmailRequest'
+      responses:
+        '200':
+          $ref: '#/components/responses/BaseSuccessResponse'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
```

Comment:
How about other error codes ? Bad request(when the input request is not valid), throttling ?

Since its an open api, we need to ensure that the request is throttled if verification email is sent for same email address more than a threshold value. Lets keep the threshold value configurable. We can keep the default as 5 emails per day for the same email addr.